.github/ISSUE_TEMPLATE/001-bug_report.yml

```diff
@@ -0,0 +1,64 @@
+name: Bug report
+description: Report an issue with Kadena
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to file a bug report. Please fill out this form as completely as possible.
+
+        ✅ I have read the [Code of Conduct](https://github.com/kadena-community/kadena.js/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
Shouldn't this be:

- [ ] I have read the [Code of Conduct](https://github.com/kadena-community/kadena.js/blob/master/CODE_OF_CONDUCT.md)
- [ ] The bug was not reported before
- [ ] I am using the **latest version** of the relevant package(s)

Instead of emojis

Also applicable in other places

Reply:
That's a great idea, it will enforce people to read and confirm; updated.